doc/developers.md

```diff
@@ -13,25 +13,25 @@ Help](https://help.github.com/articles/cloning-a-repository/).
 
 Once in the sbt shell you will be able to build and run tests for the
 connector without any Spark or Cassandra nodes running. The integration tests 
-require [CCM](https://github.com/riptano/ccm) installed on your machine. 
+require [CCM](https://github.com/riptano/ccm) installed on your machine and 
+valid path to java home set in JAVA_HOME env variable. 
 This can be accomplished with `pip install ccm`
  
 The most common commands to use when developing the connector are
 
 1. `test` - Runs the the unit tests for the project.
-2. `it:test` - Runs the integration tests with embedded Cassandra and Spark
+2. `it:test` - Runs the integration tests with Cassandra and Spark
```

Comment:
Might as well note that we run C* with CCM now

Reply:
done